doc/whats_new.rst

```diff
@@ -26,6 +26,8 @@ Bug
 
 - Fix missing autapses in network construction, by `Mainak Jas`_ in `#50 <https://github.com/jonescompneurolab/hnn-core/pull/50>`_
 
+- Fix rhythmic input feed, by 'Ryan Thorpe'_ in `#98 <https://github.com/jonescompneurolab/hnn-core/pull/98>`_
```

Comment:
oops, I merged too soon. But it's okay. Can you fix this in another PR? Your name won't appear as a link. You have to add it like the ones below.

Reply:
Oh I see, yeah I'll fix that.